projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -25,22 +25,27 @@ export const settings = {
 		const isInsideSiteEditor = document.getElementById( 'site-editor' ) !== null;
 
 		const siteFragment = getSiteFragment();
-		const launchPadUrl = getRedirectUrl( 'wpcom-launchpad-setup-link-in-bio', {
+		const launchPadUrl = getRedirectUrl( `wpcom-launchpad-setup-${ siteIntentOption }`, {
```

Comment:
nitpick, and probably not something to block this PR on. But it looks like we could set this up in a way where we don't need to re-register server side for each new intent we want to enable (see the `calypso-edit-post` example shown at the bottom of https://github.com/Automattic/jetpack/tree/trunk/projects/js-packages/components/tools/jp-redirect)

that said, im fine with that being moved to a follow up if we need to worry about it. right now we have all these source strings registered for what we want and it isnt hard to add them.